### PR TITLE
Install protoc on workflow

### PR DESCRIPTION
**Pull request by dangeross**
(cherry picked from commit 317c7906d968eeb6726ab4a9076a8bf64bdf4a6e)

### CODE REVIEW

**Review by erdemyerebasmaz — CHANGES_REQUESTED** [review]
LGTM!

One suggestion to use latest `setup-protoc` and perhaps we can upgrade the version as well if we longer need to target 23.4 specifically.

File: .github/workflows/build-bindings-android.yml [hunk 1]
```diff
@@ -49,6 +49,12 @@ jobs:
         rustup target add ${{ matrix.target }}
         cargo install cargo-ndk
 
+    - name: Install Protoc
+      uses: arduino/setup-protoc@v2
```
[the remark this answers is not in the document]
Comment (dangeross) [reply .github/workflows/build-bindings-android.yml:53]: It seems there was a reason it was fixed to 23.4 in https://github.com/breez/breez-sdk/pull/361
But it seems to be building ok, I'll bump it to v3

[the remark this answers is not in the document]
Comment (erdemyerebasmaz) [reply .github/workflows/build-bindings-android.yml]: > It seems there was a reason it was fixed to 23.4 in [breez/breez-sdk#361](https://github.com/breez/breez-sdk/pull/361) I'll bump it to v3

Hmm, hard to deduce why, please ask @JssDWt if he remembers :)

[the remark this answers is not in the document]
Comment (JssDWt) [reply .github/workflows/build-bindings-android.yml]: Trying out some updates here: https://github.com/breez/breez-sdk/pull/1027

[the remark this answers is not in the document]
Comment (JssDWt) [reply .github/workflows/build-bindings-android.yml]: Seems like protoc 27.2 is also fine. It just has to be fixed to a specific version I think.

[the remark this answers is not in the document]
Comment (erdemyerebasmaz) [reply .github/workflows/build-bindings-android.yml]: > Seems like protoc 27.2 is also fine. It just has to be fixed to a specific version I think.

Thanks for the investigation and update @JssDWt !

@dangeross let's bump `protoc` to version `27.2` and ensure all CI flows use the same version, then we can run some tests.

[the remark this answers is not in the document]
Comment (dangeross) [reply .github/workflows/build-bindings-android.yml]: Bumped to 27.2 and ran the (non-publish) publish CI. Looks good.

**Review by erdemyerebasmaz — APPROVED** [review]

**Review by ok300 — APPROVED** [review]

**Pull request merged**